CMakeLists.txt

```diff
@@ -694,8 +698,9 @@ else()
   include_directories(${CMAKE_CURRENT_BINARY_DIR}/third_party)
 endif()
 
-add_subdirectory ("${THIRD_PARTY_SOURCE_DIR}/re2_cmake" "${CMAKE_CURRENT_BINARY_DIR}/third_party/re2")
+#add_subdirectory ("${THIRD_PARTY_SOURCE_DIR}/re2_cmake" "${CMAKE_CURRENT_BINARY_DIR}/third_party/re2")
 include_directories("${THIRD_PARTY_SOURCE_DIR}/re2")
+add_subdirectory ("${THIRD_PARTY_SOURCE_DIR}/re2" "${CMAKE_CURRENT_BINARY_DIR}/third_party/re2")
```

Comment:
Removed the dead code. If I change the order of 702 and 703, it gives me following error, and hence include first and add subdirectory later. 
```
...third_party/src/re2/re2/bitstate.cc:24:26: fatal error: util/logging.h: No such file or directory
 #include "util/logging.h"
```